wrappers/android/librealsense/src/main/java/com/intel/realsense/librealsense/Pipeline.java

```diff
@@ -24,7 +24,7 @@ public void stop() {
     }
 
     public FrameSet waitForFrames() throws Exception {
-        return waitForFrames(5000);
+        return waitForFrames(5000); //defining default timeout
```

Comment:
So you can drop the value since it's the default.

Reply:
Well this is the place in which the default is assigned.
There is no API method without the timeout, so it should be assigned before calling the API method.